packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
 
 * Changed
   * Updated toggle examples for `terra-collapsible-menu-view` to include selectable toggles and toggles within a collapsed menu.
+  
+* Added
+  * Added examples and tests for Add & Close functionality of tabs.
```

Comment:
this should be moved to un-released section

Reply:
Done https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41